### PR TITLE
Migrate kubemark master size, nodes to flags

### PR DESCRIPTION
**Pull request by fejta**
/assign @krzyzacy @shyamjvs @wojtek-t 
ref https://github.com/kubernetes/test-infra/issues/3375

### CODE REVIEW

File: jobs/move_extract.py [hunk 1]
```diff
@@ -84,15 +86,6 @@ def sort():
         args = list(args)
         for key, val in new_args.items():
             args.append('%s=%s' % (key, val))
-        if not any(a.startswith('--provider=') for a in args):
```
Comment (shyamjvs) [inline jobs/move_extract.py]: Is this somehow related to kubemark or just some different change?

**Comment by shyamjvs** [comment]
The migration from env vars to flags lgtm. But I can't find equivalent changes around kubetest to set these variables as env vars. Please note that we still do need to set "KUBEMARK_MASTER_SIZE" and "KUBEMARK_NUM_NODES" as our startup scripts use them. Ref:
https://github.com/kubernetes/kubernetes/blob/master/cluster/kubemark/gce/config-default.sh#L28-L30

For now setting env var inside kubetest should do. In the long term they should move to flags but that's quite some work (as all kubemark scripts would need to be migrated to go, something similar to our e2e framework or a relatively easier alternative is to somehow add flags to all kubemark scripts inside test/kubemark).

**Comment by shyamjvs** [comment]
Sorry I just figured you're doing it here - https://github.com/kubernetes/test-infra/blob/master/kubetest/e2e.go#L475
Just that one question above and lgtm.

File: jobs/move_extract.py (hunk 1 above)
Comment (fejta) [inline jobs/move_extract.py]: Unrelated.

I just keep refactoring this file each time I migrate another env over to a flag (first was --extract, hence the name). We no longer use KUBERNTES_PROVIDER in our .env files (and have a unit test to enforce this) so the --provider migration logic is no longer necessary

Comment (shyamjvs) [inline jobs/move_extract.py]: That's cool. Thanks for explaining.

**Comment by krzyzacy** [comment]
remove `CLUSTER_IP_RANGE` in this PR? ref https://github.com/kubernetes/test-infra/pull/3590

**Comment by krzyzacy** [comment]
(the order need to be merge https://github.com/kubernetes/test-infra/pull/3670, new image, then this one)

**Comment by fejta** [comment]
CLUSTER_IP_RANGE is more complicated because it is used by both --test and --up.

**Pull request merged**